protected/humhub/modules/content/components/behaviors/CompatModuleManager.php

```diff
@@ -66,4 +66,9 @@ public function disableModule($moduleId)
     {
         return $this->moduleManager->disable($moduleId);
     }
+
+    public function isModuleAlwaysActive($id)
```

Comment:
@luke- Not directly related to this PR, but shouldn't we deprecate all functions of this class, not sure if it makes sense for behaviors regarding IDE support?

Reply:
Also I do we need to add new functions to this class, not sure which kind of modules depend on this class.

Reply:
Remove this method. It's redundant.